router/batchrouter/worker.go

```diff
@@ -28,259 +23,223 @@ func newWorker(partition string, logger logger.Logger, brt *Handle) *worker {
 		partition: partition,
 		logger:    logger,
 		brt:       brt,
+		cb: circuitbreaker.NewCircuitBreaker(
+			partition,
+			circuitbreaker.WithMaxRequests(1),
+			circuitbreaker.WithTimeout(brt.conf.GetDuration("BatchRouter.timeout", 10, time.Second)),
```

Comment:
10 seconds seems a bit on the low side, shouldn't we wait for at least uploadFreq?